changelog/next/features/5168--url-decode.md

```diff
@@ -0,0 +1,2 @@
+The `encode_url` and `decode_url` functions encode and decode URLs. For example,
+`"Hello%20World".decode_url()` returns `b"Hello World"`.
```

Comment:
Why the `b` in front of the string?

Reply:
Because it returns a blob. URL-decoding may return results that is not valid UTF-8.

Reply:
Okay, I guess it just means calling .string() again. But that's better than not being able strings to blobs.